arlo-client/src/components/Audit/Setup/Participants/index.tsx

```diff
@@ -34,9 +30,12 @@ interface IProps {
 const Participants: React.FC<IProps> = ({ audit, nextStage }: IProps) => {
   const { electionId } = useParams()
   const [isLoading, setIsLoading] = useState(false)
+  const [{ state = '' }, updateState] = useAuditSettings(electionId!)
```

Comment:
Naming this `updateState` implies that you can call `updateState(state)`, but you can't. Just call it `updateSettings`?

Reply:
I waffled over the naming here a bit, yeah. Good point.